extra/mariabackup/xtrabackup.cc

```diff
@@ -1836,7 +1842,11 @@ GNU General Public License for more details.\n\
 You can download full text of the license on http://www.gnu.org/licenses/gpl-2.0.txt\n");
 
   printf("Usage: %s [--defaults-file=#] [--backup | --prepare | --copy-back | --move-back] [OPTIONS]\n",my_progname);
-  print_defaults("my", load_default_groups);
+  std::vector<const char*> backup_load_default_groups;
+  concatenate_default_groups(backup_load_default_groups, load_default_groups);
+  concatenate_default_groups(backup_load_default_groups, backup_default_groups);
```

Comment:
In order to have the same output as in [documentation](https://mariadb.com/kb/en/mariabackup-overview/#server-option-groups) where backup options are output maybe to use them before `load_default_groups`.